dialer_test.go

```diff
@@ -195,16 +207,18 @@ func TestConnectState(main *testing.T) {
 
 		dialer, err := amqpextra.NewDialer(
 			amqpextra.WithURL("amqp://rabbitmq.host"),
-			amqpextra.WithAMQPDial(amqpDialStub(time.Millisecond*150, conn)),
+			amqpextra.WithAMQPDial(amqpDialStub(conn)),
```

Comment:
Revert `time.Millisecond*150`.  The behavior is different without it

Reply:
+